Multisig/App/RemoteNotificationHandler.swift

```diff
@@ -163,6 +173,34 @@ class RemoteNotificationHandler {
         Tracker.setPushInfo(status.trackingStatus.rawValue)
     }
 
+
+    static func signAsync(safes: [String], deviceID: String, token: String, timestamp: String) async throws -> (preimage: String, hash: String, signatures: [String]) {
+
+        // get keys for signing
+        var privateKeys: [PrivateKey] = []
+        let keyInfos = try KeyInfo.all()
+        for keyInfo in keyInfos {
+            let privateKey = try await keyInfo.pushNotificationSigningKey()!
+            privateKeys.append(privateKey)
+        }
+
+        let hashPreimage = [
+            "gnosis-safe",
```

Comment:
I wonder if this should stay gnosis-safe or we can rebrand to "safe only

Reply:
backend has to be adjusted also. Otherwise signature check will fail. I would not touch this now.